include/couch_db.hrl

```diff
@@ -91,10 +91,27 @@
     design_url_handlers,
     auth,
     default_fun,
-    url_handlers
+    url_handlers,
+    authentication_handlers = [],
+    absolute_uri,
+    auth_module,
+    begin_ts,
+    original_method,
+    nonce,
+    cors_config,
+    qs
+}).
+
+-record(httpd_res, {
```

Comment:
I can move it. However this would mean that the plugin developer would need to do:
```
-include_lib("chttpd/include/chttpd.hrl").
```
Which is alright.